docs/user_guide/api/LudwigModel.md

```diff
@@ -487,6 +488,11 @@ Using a trained model, make predictions from the provided dataset.
   to use. Possible values are `'full'`, `'training'`, `'validation'`, `'test'`.
 - **batch_size** (int, default: 128): size of batch to use when making
   predictions.
+- **generation_config** (Dict, default: `None`) config for the generation of the
+  predictions. If `None`, the config that was used during model training is
+  used. This is only used if the model type is LLM. Otherwise, this parameter is
+  ignored. See [Large Language Models](../../configuration/large_language_model.md) under "Generation" for
```

Comment:
Heads up: this style of linking (unfortunately) doesn't work when the website is built and deployed by `mike`. The best solution I've found is to just use the relative link style (`/latest/configuration/large_language_model/`).

Reply:
If you want to replicate and verify, follow the instructions in the [Versioned Docs section of the README](https://github.com/ludwig-ai/ludwig-docs#versioned-docs).

Reply:
Thanks Martin. I've updated the link